src/BuildPrediction/Predictors/ReferenceItemsPredictor.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Build.Prediction.Predictors
+{
+    using System;
+    using System.IO;
+    using Microsoft.Build.Evaluation;
+    using Microsoft.Build.Execution;
+
+    /// <summary>
+    /// Finds EmbeddedResource items as inputs.
+    /// </summary>
+    public class ReferenceItemsPredictor : IProjectPredictor
```

Comment:
>ReferenceItemsPredictor [](start = 17, length = 23)

Not added to top-level predictor list and UT

Reply:
Damnit! Maybe I do need a reflection UT for that...